### DISCUSSION TITLE
Raman Intensity?

### DISCUSSION THREAD

**Discussion by attlbende** [grimme-lab/xtb#639] [Q&A]
Dear Developers,

Is it possible to compute Raman intensities together with the Frequency analysis using the GFN-xTB2 method?

Many thanks
Attila

**Comment by awvwgk** [comment]
Computing Raman intensities requires access to the polarizability tensor during the finite difference calculation of the hessian, which is currently not available since it is a response property. A surrogate could be the D4 isotropic polarizibility, however this is expected to provide insufficient accuracy. Even with the polarizability tensor of GFN2-xTB the resulting Raman intensities are expected to be poor as only a rather small basis set is used for all xTB Hamiltonians.
